index.yaml

```diff
@@ -12,13 +12,13 @@ entries:
       catalog.cattle.io/release-name: fleet
     apiVersion: v2
     appVersion: 0.3.6-rc2
-    created: "2021-06-16T21:05:17.210021-07:00"
+    created: "2021-06-17T22:33:22.888322-07:00"
     dependencies:
     - condition: gitops.enabled
       name: gitjob
       repository: file://./charts/gitjob
     description: Fleet Manager - GitOps at Scale
-    digest: e0851254ef96f419c2d2bd0976ffe1651e85b57a3a3c895ae83c7b3df972e09f
+    digest: 2ff33c8499e9b05ffe9b99f40865810356aeae447b446914e781ddceea8db0a4
```

Comment:
@aiyengar2  I do not understand why this digest value and a lot of them in the rest of the file is changed. there should be nothing changed for this `assets/fleet/fleet-100.0.0+up0.3.6-rc2.tgz` file. I guess it is related to the fact that git marks `assets/fleet/fleet-100.0.0+up0.3.6-rc2.tgz`  deleted? 

Another interesting finding from my experiments: I got a different value for this version of fleet if I delete charts/, assets/ and index.yaml, then run make validation twice in the dev-v2.6 branch. 

I doubt if it is a bug in the charts-build-scripts, but I have not find any evidence.